src/components/application_manager/test/resumption_sql_queries_test.cc

```diff
@@ -84,7 +84,9 @@ const string kDeleteData =
     "COMMIT; "
     "VACUUM;";
 
-const std::string kJournalOff = "PRAGMA journal_mode = OFF;";
+const static std::string kJournalOff = "PRAGMA journal_mode = OFF;";
```

Comment:
@dev-gh May be just use nameless namespace without adding `static` qualifier? I mean to wrap these strings.